components/BridgeUnclaimedWithdrawals.tsx

```diff
@@ -0,0 +1,145 @@
+import { useLayoutEffect, useState, VFC } from "react";
+import { css } from "@emotion/react";
+import {
+	LinearProgress,
+	Table,
+	TableBody,
+	TableCell,
+	TableContainer,
+	TableHead,
+	TableRow,
+	Theme,
+} from "@mui/material";
+import { useHistoricalWithdrawRequest } from "@/hooks";
+import { getMinutesAndSeconds } from "@/utils";
+import { useBridge } from "@/providers/BridgeProvider";
+
+const _renderExpiry = (expiryRaw, expiryString) => {
+	const [expiry, setExpiry] = useState<string>("");
+	const [seconds, setSeconds] = useState<number>(0);
+
+	useLayoutEffect(() => {
+		if (expiryRaw * 1000 > Date.now() + 600000) return setExpiry(expiryString);
+
+		const intervalId = setInterval(() => {
+			setSeconds((seconds) => seconds + 1);
+			setExpiry(getMinutesAndSeconds(expiryRaw - seconds));
+		}, 1000);
+
+		return () => clearInterval(intervalId);
+	}, [expiryString, expiryRaw, seconds, setExpiry]);
+
+	return (
+		<TableCell css={[styles.column, styles.number]}>
+			{expiry && expiry}
+			{!expiry && <LinearProgress css={[styles.expiryProgress]} />}
+		</TableCell>
+	);
+};
+
+const BridgeUnclaimedWithdrawals: VFC = () => {
+	const { unclaimedWithdrawals } = useBridge();
+	const processHistoricalRequest = useHistoricalWithdrawRequest();
+
+	return (
+		<TableContainer css={styles.container}>
+			<Table>
+				<TableHead>
+					<TableRow>
+						<TableCell css={styles.column}>ID</TableCell>
+						<TableCell css={styles.column}>Value</TableCell>
+						<TableCell css={styles.column}>Expiry</TableCell>
+						<TableCell css={styles.column}>Action</TableCell>
+					</TableRow>
+				</TableHead>
+				<TableBody>
+					{unclaimedWithdrawals?.map((unclaimed) => (
+						<>
+							{unclaimed.expiry !== "Expired" && (
+								<TableRow css={styles.row} key={unclaimed.eventProofId}>
+									{/* ID */}
+									<TableCell css={[styles.column, styles.number]}>
+										{unclaimed.eventProofId}
+									</TableCell>
+									{/* Value */}
+									<TableCell css={[styles.column, styles.number]}>
+										{unclaimed.transferAmount.toInput()}{" "}
+										{unclaimed.transferAsset.symbol}
+									</TableCell>
+									{/* Expiry */}
+									{_renderExpiry(unclaimed.expiryRaw, unclaimed.expiry)}
```

Comment:
let's convert `_renderExpiry` into an `ExpiryCell` (can be in the same file) that receive 2 props